doc/host_config_schema/cchost_config.json

```diff
@@ -467,6 +467,48 @@
       },
       "description": "This section includes configuration for the host-enclave ring-buffer memory (modify with care!)",
       "additionalProperties": false
+    },
+    "acme_client_config": {
+      "type": "object",
+      "properties": {
+        "ca_cert": {
+          "type": "string",
+          "default": "",
+          "description": "Certificate of the ACME server we will connect to (to avoid certificate verification failures)"
+        },
+        "directory_url": {
+          "type": "string",
+          "default": "",
+          "description": "URL of the ACME server's directory"
+        },
+        "service_dns_name": {
+          "type": "string",
+          "default": "",
+          "description": "DNS name of the service we represent"
+        },
+        "contact": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Contact address(es)"
```

Comment:
Email?

Reply:
It's more complicated than that, I'll expand.